### ISSUE TITLE
The plugin has a typo in the name

### ISSUE THREAD

**Issue by mc-perry** [andrehtissot/cordova-plugin-fcm-with-dependecy-updated#60]
The name of the plugin should be
cordova-plugin-fcm-with-dependency-updated

The current name is kind of unprofessional. Should be a really simple change.

**Comment by andrehtissot** [comment]
I agree. And to be honest, this started as fork for my own use, which had only the dependencies updated. But due to bugs I found and customisations I wanted, it grew, not with intention of being used by others. And so, the name stuck.

It is not an easy change, neither here or on the npm, as many links use the old name.
What I can do is to deprecate this repository/package and point to a new one with a better name. Which is kind of bother for the users.

**Comment by andrehtissot** [comment]
It should be done, I agree.
